### PR TITLE
aws.md

### PR DESCRIPTION
**Pull request by zhengyili4321**

### CODE REVIEW

File: chapters/iaas/aws/aws.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Amazon in the following document:
 * <https://d0.awsstatic.com/whitepapers/aws-overview.pdf>
 
 We list the product in screenshots from their Product Web page panel
-in Figure next: @fig:aws-products-1, @fig:aws-products-2.
+in next Figure: @fig:aws-products-1, @fig:aws-products-2.
```
Comment (bfeng) [inline chapters/iaas/aws/aws.md]: As figures are specified with labels, we don't need the word "next" here.

File: chapters/iaas/aws/aws.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ Within each category you have several products. When choosing products
 form AWS it is best to start with the overview paper and identify
 products that can be of benefit to you. For our purpose we focus on
 the traditional Compute and Storage offerings. We list the products
-that are available In Sep. 2018 next.
+that are available In Sep. 2018 below.
```
Comment (bfeng) [inline chapters/iaas/aws/aws.md]: Why changed to below?

File: chapters/iaas/aws/aws.md [hunk 3]
```diff
@@ -294,7 +294,7 @@ To go to the EC2 services you can click on the services link on the
 top left corner in the console and then click on EC2 which is listed
 under "Compute". Then you will see a blue button labeled "Launch
 instance". Click on the button and the console will take you to the
-page shown next: @fig:launch-instance. Notice that the check box for "Free tier only" is
+page shown below: @fig:launch-instance. Notice that the check box for "Free tier only" is
```
Comment (bfeng) [inline chapters/iaas/aws/aws.md]: ?

File: chapters/iaas/aws/aws.md (hunk 1 above)
Comment (zhengyili4321) [reply chapters/iaas/aws/aws.md]: ok. I will delete all 'next' as long as the figure label is shown up

File: chapters/iaas/aws/aws.md (hunk 2 above)
Comment (zhengyili4321) [reply chapters/iaas/aws/aws.md]: I will change it back

**Pull request merged**